google_guest_agent/diagnostics.go

```diff
@@ -60,32 +58,8 @@ func (d *diagnosticsMgr) timeout() bool {
 }
 
 func (d *diagnosticsMgr) disabled(os string) (disabled bool) {
```

Comment:
All features we have ways to disable, i dont think you should take out the metadata or config options.

Reply:
Is there a reason that this should be disable-able, other than other features of the agent can be disabled?

It seems harmless to leave this enabled and having to activate it before using the gcloud command seems like an annoyance (and some customers agree, from PR description), but I'm no expert.

Reply:
How about taking out of the metadata options and leave the config options enabled by default?

Reply:
@adjackura Can you take a look at my latest change? The config options is there, so we can disable it.

Reply:
Making agent features disablable via metadata is pretty important for customers that want to restrict what users can instruct the agent to do. Enabled by default is fine, and very few users ever disable the features, but all other features have clear ways to disable so this should match those.

Reply:
Ok, leave all the config & metadata options, just enable it by default if not defined.